src/Element/NodeElement.php

```diff
@@ -118,6 +118,18 @@ public function hasAttribute($name)
     }
 
     /**
+     * Checks whether element does not have an attribute with specified name.
+     *
+     * @param string $name
+     *
+     * @return Boolean
+     */
+    public function doesNotHaveAttribute($name)
```

Comment:
-1 for this method. Use ``!$element->hasAttribute($name)`` in the WebAssert class